net/instaweb/rewriter/add_instrumentation_filter.cc

```diff
@@ -95,7 +101,12 @@ void AddInstrumentationFilter::AddHeadScript(HtmlElement* element) {
     added_head_script_ = true;
     // TODO(abliss): add an actual element instead, so other filters can
     // rewrite this JS
-    HtmlCharactersNode* script = driver()->NewCharactersNode(NULL, kHeadScript);
+    HtmlCharactersNode* script = NULL;
+    if(driver()->options()->Enabled(RewriteOptions::kPedantic)){
+        script = driver()->NewCharactersNode(NULL, kHeadScriptPedantic);
```

Comment:
s/NULL/nullptr/ in new code.  The codebase was written pre-cxx11 so there are some NULLs in this file.  It's up to you if you want to fix those while you are in here.